### PR TITLE
Fix for Issure #18933: change "install media" to "boot media" and add more detailed descript…

### PR DESCRIPTION
**Pull request by hiveai**
…ion for x86 installation

## Proposed change

Change "installation media" to target "boot media" and clarify the instructions for x86, since to install HAOS, you do not create an install media, you write the disk image onto the target "boot media" 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- This PR fixes or closes issue: #18933 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @hiveai, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18934).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -69,15 +73,15 @@ To boot Home Assistant OS the BIOS needs to have UEFI boot mode enabled and Secu
...
-If you prefer to use a live operating system, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, the following steps on how-to write the image to your installation media can be followed.
+If you prefer to use a live operating system, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, the following steps on how to write the image to your boot media can be followed.
 
 {% endif %}
 
-### Write the image to your installation media
+### Write the image to your home assistant boot media
 
-1. Attach the installation media ({{site.installation.types[page.installation_type].installation_media}}) to your computer
+1. Attach the home assistant boot media ({{site.installation.types[page.installation_type].installation_media}}) to your computer
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]: Home Assistant is a name and needs to be proper capitalized.

```suggestion
1. Attach the Home Assistant boot media ({{site.installation.types[page.installation_type].installation_media}}) to your computer
```

File: source/_includes/installation/operating_system.md [hunk 2]
```diff
@@ -69,15 +73,15 @@ To boot Home Assistant OS the BIOS needs to have UEFI boot mode enabled and Secu
...
 
-If you prefer to use a live operating system, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, the following steps on how-to write the image to your installation media can be followed.
+If you prefer to use a live operating system, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, the following steps on how to write the image to your boot media can be followed.
 
 {% endif %}
 
-### Write the image to your installation media
+### Write the image to your home assistant boot media
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]: I don't think we need to add "Home Assistant" again here.

```suggestion
### Write the image to your boot media
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_includes/installation/operating_system.md (hunk 2 above)
Comment (hiveai) [reply source/_includes/installation/operating_system.md]: I made the suggested changes and pushed a new version

File: source/_includes/installation/operating_system.md (hunk 1 above)
Comment (hiveai) [reply source/_includes/installation/operating_system.md]: I made the suggested changes and pushed a new version

**Review by frenck — APPROVED** [review]
Thanks, @hiveai 👍

**Pull request merged**